src/coreclr/jit/inline.def

```diff
@@ -80,6 +80,7 @@ INLINE_OBSERVATION(HAS_PINNED_LOCALS,         bool,   "has pinned locals",
 INLINE_OBSERVATION(HAS_SIMD,                  bool,   "has SIMD arg, local, or ret",          INFORMATION, CALLEE)
 INLINE_OBSERVATION(HAS_SWITCH,                bool,   "has switch",                           INFORMATION, CALLEE)
 INLINE_OBSERVATION(IL_CODE_SIZE,              int,    "number of bytes of IL",                INFORMATION, CALLEE)
+INLINE_OBSERVATION(INTRINSIC_CALL,            bool,   "intrinsic call",                       INFORMATION, CALLEE)
```

Comment:
Do you think its worth tracking the named intrinsic (ni) rather than just a `bool`?

That would let us differentiate between just a JIT intrinsic vs something more special, like `HWIntrinsic` or the `Math` intrinsics (which always compile down to a single instruction when supported).

Reply:
In particular, since we have the intrinsic ID we can know during observation time if an intrinsic is or is not supported, and they will always end up at least 1 byte smaller in codegen than a call token when supported (on x86/x64), but without the overhead of callee save/restore, without the overhead of needing to push around registers, and without the overhead of the actual call as well.

Reply:
Could be we'll want something more general. For now, this mainly will unblock methods already marked with AggressiveInlining -- the changes to the discretionary heuristics likely don't help much.